edxval/tests/test_api.py

```diff
@@ -1328,7 +1327,7 @@ def test_video_transcript_missing_attribute(self, mock_logger):
             <video_asset>
                 <transcripts>
                     {transcript_xml}
-                    <transcript file_name="wow.sjson" language_code="de" file_format="sjson" provider='3PlayMedia' video_id="{video_id}"/>
+                    <transcript file_name="edxval/tests/data/wow.sjson" language_code="de" file_format="sjson" provider='3PlayMedia' video_id="{video_id}"/>
```

Comment:
why changes in `file_name` attribute above?

Reply:
As that's changed in data dictionary `VIDEO_TRANSCRIPT_3PLAY`